cmd/flexctl/backups.go

```diff
@@ -224,3 +227,126 @@ func listBackups(cmd *cobra.Command) error {
 func backupsEnabled() bool {
 	return os.Getenv("S3_ARCHIVE_CONFIG") != ""
 }
+
+func newBackupConfigCmd() *cobra.Command {
+	var configCmd = &cobra.Command{
+		Use:   "config",
+		Short: "Manage backup configuration",
+	}
+
+	configCmd.AddCommand(newConfigShowCmd())
+	configCmd.AddCommand(newConfigUpdateCmd())
+
+	return configCmd
+}
+
+type configShowResult struct {
+	Result flypg.BarmanSettings `json:"result"`
+}
+
+func newConfigShowCmd() *cobra.Command {
+	var configShowCmd = &cobra.Command{
+		Use:   "show",
+		Short: "Show current configuration",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			resp, err := http.Get("http://localhost:5500/commands/admin/settings/view/barman")
+			if err != nil {
+				return err
+			}
+
+			var rv configShowResult
+			err = json.NewDecoder(resp.Body).Decode(&rv)
+			if err != nil {
+				return err
+			}
+
+			fmt.Printf("  ArchiveTimeout = %s\n", rv.Result.ArchiveTimeout)
```

Comment:
To future proof this, we could consider just pretty printing the json output.  However, I think this is totally fine for now.

Reply:
I went back and forth on this but came to the same conclusion. If we make it more unwieldy, I'm happy to switch it later, or if you think we should do it now then I can.